includes/admin/load.php

```diff
@@ -330,8 +330,24 @@ static function ( $name ) {
 	}
 
 	if ( isset( $_GET['activate'] ) ) { // phpcs:ignore WordPress.Security.NonceVerification.Recommended
+		$message = __( 'Feature activated.', 'performance-lab' );
+
+		$plugin_settings_link = perflab_get_plugin_settings_link( $_GET['activate'] ); // phpcs:ignore WordPress.Security.NonceVerification.Recommended
```

Comment:
Technically since all input vars are slashed in WP, this should get unslashed (even though it is highly unlikely to ever have slashes):

```suggestion
		$plugin_settings_link = perflab_get_plugin_settings_link( wp_unslash( $_GET['activate'] ) ); // phpcs:ignore WordPress.Security.NonceVerification.Recommended
```